Foundation/include/Poco/Dynamic/Var.h

```diff
@@ -88,12 +88,19 @@ class Foundation_API Var
 	Var();
 		/// Creates an empty Var.
 
-	template <typename T>
+	template <typename T, std::enable_if_t<!std::is_enum<T>::value, bool> = true>
 	Var(const T& val)
 		/// Creates the Var from the given value.
 	{
 		construct(val);
 	}
+	
+	template <typename T, std::enable_if_t<std::is_enum<T>::value, bool> = true>
+	Var(const T& val)
+		/// Creates the Var from the given enum value trough cast to int
+	{
+		construct(static_cast<int>(val));
```

Comment:
I think that it would make sense to use std::undelying_type when casting. An enum can be declared with specific integer type.

https://en.cppreference.com/w/cpp/types/underlying_type